src/Util.php

```diff
@@ -67,7 +67,7 @@ public static function escapeDateMath($requestUri)
 
         // '\\{' and '\\}' should not be escaped
         if (\str_contains($uriSegment, '\\\\')) {
-            $escapedUriSegment = \str_replace(['\\\\%7B', '\\\\%7D'], ['\\\\{', '\\\\}'], $escapedUriSegment);
+            $escapedUriSegment = \str_replace(['\\\%7B', '\\\%7D'], ['\\\{', '\\\}'], $escapedUriSegment);
```

Comment:
Can you help me understand this change and the one below?

Reply:
Probably I could explain better than described in the https://mlocati.github.io/php-cs-fixer-configurator/#version:3.59|fixer:string_implicit_backslashes.

I decided to disable this rule so code stay as it was before.

Reply:
thx